src/lib.rs

```diff
@@ -485,52 +520,36 @@ impl UzfsTestEnv {
     /// If dsname is not empty, this function will create a temp file as block device, the size of device
     /// is `dev_size` bytes.
     ///
-    /// A zpool and dataset then are created upon the block device, the name is specified by `dsname`.
-    ///
     /// If dsname is empty, no block device and pool/dataset is created
     ///
     /// A temp cache file is always automacally created, used by uzfs to find zpool.
     ///
-    /// All of the resources (temp file, zpool, dataset) will be deleted automatically when the env
+    /// All of the resources (temp file, dev_file) will be deleted automatically when the env
     /// goes out of scope
     pub fn new(dsname: String, dev_size: u64) -> Self {
```

Comment:
Looks we can simplify UzfsTestEnv further:

Typically there are two kinds of usages for UzfsTestEnv:

1. setup a tmp cache file, and deleted automatically when env goes out of scope. this is needed by almost all sfs tests, because we don't want to different test use the same default cache file, which is /tmp/zpool.cache.
2. setup a tmp dev file, and deleted automatically when env goes out of scope. this is needed by some sfs tests, e.g. in meta_store test. Keep in mind that, some tests doesn't need the dev file, one example is meta_shard_manager test.

So we can remove the dsname parameter, and only retain the dev_size parameter. if dev_size == 0, then we only setup the tmp cache file, otherwise, we also setup the dev file.